src/CodeGen_LLVM.cpp

```diff
@@ -2475,7 +2475,23 @@ void CodeGen_LLVM::visit(const Call *op) {
         internal_assert(op->args.size() == 3);
         Expr cond = op->args[0];
         if (cond.type().is_vector()) {
-            scalarize(op);
+            // This will dramatically slow down the code, so it's worthy
```

Comment:
I wouldn't bother with this warning. We already have lots of things that silently scalarize and have bad performance, and this warning is going to be mostly ignored.

Reply:
Done